Bottom: 07352998882094ad56f8e446112bac932e333bae
Top:    9fbd5a9eb19f822e1fd393f1d441c7b75aa9b605
Author: Parth Shah <parth@linux.ibm.com>
Date:   2020-01-09 12:06:51 +0530

Refresh of sched-core-add-manual

---

diff --git a/kernel/sched/core.c b/kernel/sched/core.c
index 42b3c4c5559c..e5533eecca9e 100644
--- a/kernel/sched/core.c
+++ b/kernel/sched/core.c
@@ -5403,9 +5403,6 @@ SYSCALL_DEFINE4(sched_getattr, pid_t, pid, struct sched_attr __user *, uattr,
 	kattr.sched_util_max = p->uclamp_req[UCLAMP_MAX].value;
 #endif
 
-	if (p->flags & PF_CAN_BE_PACKED)
-		attr.sched_flags |= SCHED_FLAG_TASK_PACKING;
-
 	rcu_read_unlock();
 
 	return sched_attr_copy_to_user(uattr, &kattr, usize);
